x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -43,6 +45,7 @@ export const SessionView = ({ sessionEntityId, height, jumpToEvent }: SessionVie
 
   const [searchQuery, setSearchQuery] = useState('');
   const [searchResults, setSearchResults] = useState<Process[] | null>(null);
+  const [optionsStates, setOptionsStates] = useState({ timeStamp: true, verboseMode: true });
```

Comment:
Should make this use the `StateField` interface declared in `x-pack/plugins/session_view/public/components/session_view_toggle_options/index.tsx`